shortlister/controller.py

```diff
@@ -129,6 +136,24 @@ def clear_score(self, k=None):
         self.view_applicant_details()
         self.options = self.options_applicant_detail
     
+        self.view_applicant_details()    
+    
+    def sort(self,k=None):
+        """Activates sort"""
+        if k == "S":  #activate sort
+            print("Entering sorting mode")
+            self.options = self.options_sort
+        elif k == "a":
+            sort_alpha(self.shortlist.applicants)
+            self.show_applicants_list()
+        elif k == "s":
+            sort_ascending_score(self.shortlist.applicants)
+            self.show_applicants_list()
+        elif k == "d":
+            sort_descending_score(self.shortlist.applicants)
+            self.show_applicants_list()
```

Comment:
As you're calling `show_applicants_list` once sorted, you can pull that out and put outside the condition but you'll need to put a `return` (i.e. exit the function) after the "S" keypress, so it doesn't continue executing the function (otherwise "S" will also show the applicant list)

```suggestion
        if k == "S":  #activate sort
            print("Entering sorting mode")
            self.options = self.options_sort
            return
        elif k == "a":
            sort_alpha(self.shortlist.applicants)
        elif k == "s":
            sort_ascending_score(self.shortlist.applicants)
        elif k == "d":
            sort_descending_score(self.shortlist.applicants)
            
        self.show_applicants_list()
```